tests/js-tests/src/CocosDenshionTest/CocosDenshionTest.js

```diff
@@ -231,7 +231,11 @@ var soundId = null;
 
 var playMusic = function () {
     cc.log("play background music");
-    audioEngine.playMusic(MUSIC_FILE, false);
+    var musicFile = MUSIC_FILE;
+    if (cc.sys.os == cc.sys.OS_ANDROID) {
+        musicFile = "res/"+musicFile;
+    }
+    audioEngine.playMusic(musicFile, false);
```

Comment:
@minggo These png file path is used in C/C++ layer, it have extra process to get right path, but in CocosDenshionTest, these param will directly send to Java layer, which not have search path or similar process to find full path of the file. Please refer to Java layer implementation at https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxMusic.java#L244.